.babelrc.js

```diff
@@ -43,7 +44,7 @@ const config = {
 			{ async: isCalypsoClient && codeSplit },
 		],
 		'@babel/plugin-proposal-export-default-from',
-		'@babel/transform-runtime',
+		[ '@babel/transform-runtime', { regenerator: false } ],
```

Comment:
I'm not entirely sure this is necessary, but it's included since @blowery mentioned something about disabling the `regenerator transform` & I wanted to make sure to capture that suggestion.